scanners/dmarc/dmarc_scanner.py

```diff
@@ -107,16 +64,44 @@ def scan(scan_id, domain):
         scan_result = json.loads(json.dumps(check_domains(domain_list, skip_tls=True)))
     except (DNSException, SPFError, DMARCError) as e:
         logging.error(
-            "(SCAN: %s) - Failed to check the given domains for DMARC/SPF records: %s"
-            % (scan_id, e)
+            "Failed to check the given domains for DMARC/SPF records: %s"
+            % str(e)
         )
         return None
 
-    if scan_result["dmarc"]["record"] is "null":
+    if scan_result["dmarc"]["record"] == "null":
```

Comment:
Just wondering if this value is always available to retrieve from, may be a good idea to switch to `.get()` just in case it cannot be found.

Reply:
Good catch. Updated that in the event it isnt.